File: C_Programing/Competitive/Problems_Solving/ASSIGNMENT_15/Q1.c
//WAP accept n  numbers from user and accept onr another number as NO,check whether No is present or not..


#include<stdio.h>
#include<stdlib.h>

#define TRUE 1
#define FALSE 0

typedef int BOOL;

BOOL Check(int Arr[],int iLength,int iNo)
{
  int iCnt=0;
  
  for(iCnt=0;iCnt<iLength;iCnt++)
  {
    if(Arr[iCnt]==iNo)
    {
        return TRUE;
    }
    
  }
  return FALSE;

}


int main()
{
    int iCnt=0;
    int iValue=0;
    int iSize=0;
    int *p=NULL;
    BOOL bRet=FALSE;

    printf("Enter Number of Elements :\n ");
    scanf("%d",&iSize);


    p=(int*)malloc(iSize*sizeof(int));

     
    if(p==NULL)
     {
        printf("Unable to allocte the memory\n");
        return-1;

      }
      printf("Enter %d Elements\n",iSize);

      for(iCnt=0;iCnt<iSize;iCnt++)
        {
            printf("\nEnter Element :%d\n",iCnt+1);
            scanf("%d",&p[iCnt]);

        }
            

     printf("Enter another Number:\n");
             scanf("%d",&iValue);
        bRet=Check(p,iSize,iValue);

        if(bRet==TRUE)
        {
            printf("Number is Present\n ");
 
        }
        else
        { 
             printf("Number is Absent\n");
             
        }
        free(p);

        return 0;



}

File: java_Programing/Competitive/Problems_Solving/ASSIGNMENT_18/Q1.java

import java.util.Scanner;


//WAP to check whether a number is prime or not

class Logic
{
    void CheckPrime(int NO)
    {
          int iCnt = 0;

        if(NO <= 1)
        {
            System.out.println(NO + " is not a Prime Number");
            return;
        }
        
        for(iCnt = 2; iCnt * iCnt <= NO; iCnt++)
        {
            if(NO % iCnt == 0)
            {
                System.out.println(NO + " is not a Prime Number");
                return;
            }
        }

        System.out.println(NO+ " is a Prime Number");
    }
}




class Q1
{
    
    public static void main(String A[])
    {
           int iValue =0;

        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter Random Number :");

        iValue=sobj.nextInt();

        Logic lobj=new Logic();

        lobj.CheckPrime(iValue);
    }

}